README.md

```diff
@@ -133,3 +133,60 @@ var intValue: Int
 var intValue : Int
 
 ```
+
+### Enums ###
+
+For enum declarations, declare each enum case on a new line with its own `case` statement instead of a comma-separated list.
+
+```swift
+internal enum State
```

Comment:
This is consistent with our rule for always specifying access modifiers at the top-level. In this example, the enum is assumed top-level and is getting an internal access modifier.